translator/__init__.py

```
@@ -30,38 +30,70 @@ class Translator(object):
     def __add__(self, other):
+        """Add two translators together using the + operator.
+
+        Parameters:
+            other: A Translator object to append.
+
+        Returns:
+            A new Translator combining self and other.
+        """
         return self.append(other)
 
     def __iadd__(self, other):
```

Ruff (0.14.5)

Missing return type annotation for special method `__iadd__`

(ANN204)